spec/controllers/system/admin/users_controller_spec.rb

```diff
@@ -7,14 +7,10 @@
     let(:admin) { create(:administrator) }
     let(:instance_admin) do
       user = create(:user)
-      create(:instance_user, instance: instance, user: user, role: :administrator)
-      user
-    end
-    let(:normal_user) do
-      user = create(:user)
-      create(:instance_user, instance: instance, user: user, role: :normal)
+      user.instance_users.first.update_attributes(role: :administrator)
```

Comment:
would it make sense here to create(:instance_user, role: administrator).user instead?